lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,10 @@
       }
     }
   }
+
+  @media print {
```

Comment:
If you just want to get down to a copyright notice, then that's straightforward. When Sarina wanted the course name as well, that required digging into the template.

I'll make some changes to keep a small footer present.